src/NLog/LogFactory.cs

```diff
@@ -1348,7 +1350,7 @@ private IEnumerable<Logger> GetLoggers()
                 List<Logger> values = new List<Logger>(loggerCache.Count);
 
                 //new list for prevent InvalidOperationException on Travis/Mono.
```

Comment:
@jpdillingham If removing the comment, then you should also remove the ToList()-operation.